composer.json

```diff
@@ -14,7 +14,11 @@
 		"laravel/framework": "^11.0",
 		"laravel/sanctum": "^4.0",
 		"laravel/tinker": "^2.9",
+		"spatie/data-transfer-object": "^3.9",
```

Comment:
Do we still need this dependency?

Reply:
It is listed as a required package on the packagist for `spatie/typescript-transformer`. I'm not sure if that's a hard requirement or just something packagist recommends?